src/avx2.c

```diff
@@ -2,6 +2,14 @@
 
 #if SZ_USE_X86_AVX2
 #include <x86intrin.h>
+#include <stdint.h>
+
+typedef union {
+    uint64_t u64;
+    uint32_t u16s[sizeof(uint64_t) / sizeof(uint32_t)];
+    uint16_t u32s[sizeof(uint64_t) / sizeof(uint16_t)];
+    uint8_t u8s[sizeof(uint64_t) / sizeof(uint8_t)];
+} u64_parts_t;
```

Comment:
The `src/` folder was a temporary decision during the v3 preparation. I'll remove it in a bit to avoid confusion. Sorry, I got a bit used to working on this alone :)